csharp/src/sdk/Geometry/Polygon.cs

```diff
@@ -322,41 +224,190 @@ internal static bool Touches(Polygon p1, Polygon p2)
         }
 
         /// <summary>
-        /// Performs difference,intersection, union, or XOR operations on two polygons.
+        /// Returns a list of Polygons representing the difference between this Polygon and the supplied Polygon.
```

Comment:
See my comment above on using the `<returns>` tag. In this instance, you'd also want the returns tag to specify what happens when there is no difference result. Do we get an empty list, null, does it throw an exception? In this case, the returns would say: "Returns a collection of Polygons representing the difference between this Polygon and the supplied polygon. If the provided Polygons do not overlap, this returns an empty collection."

Reply:
The returns tag can also be valuable to hint at whether the method changes the internal state of something or returns a new thing. This doesn't apply in this instance, but may in others.